github_backup/github_backup.py

```diff
@@ -357,18 +361,23 @@ def get_auth(args, encode=True, for_git_cli=False):
                 raise Exception('No password item matching the provided name and account could be found in the osx keychain.')
     elif args.osx_keychain_item_account:
         raise Exception('You must specify both name and account fields for osx keychain password items')
-    elif args.token:
+    elif args.token_fine:
+        if args.token_fine.startswith("github_pat_"):
+            auth = args.token_fine
+        else:
+            raise Exception("Fine-grained token supplied does not look like a GitHub PAT")
+    elif args.token_classic:
         _path_specifier = 'file://'
-        if args.token.startswith(_path_specifier):
-            args.token = open(args.token[len(_path_specifier):],
-                              'rt').readline().strip()
+        if args.token_classic.startswith(_path_specifier):
+            args.token_classic = open(args.token_classic[len(_path_specifier):],
+                                      'rt').readline().strip()
```

Comment:
need to support this ( `file://` ) for fine grained tokens too. otherwise you have to include the PAT on the command line.